openedx/adg/lms/applications/helpers.py

```diff
@@ -865,3 +872,40 @@ def send_application_status_update_email(application, message_for_applicant):
     task_send_mandrill_email.delay(
         status_email_template_map[application.status], [applicant.email], email_context
     )
+
+
+def check_and_update_prereq_courses_status_to_completed(user_id, course_id):
+    """
+    Update prereq_courses_status to COMPLETED in Application Hub
+    depending on the prereq courses that the learner has enrolled in
+    """
```

Comment:
Our first thing in this method should be 

if is_testing_environment:
    return